db/migrate/20161114122001_create_tier_and_service_tier.rb

```diff
@@ -0,0 +1,76 @@
+class CreateTierAndServiceTier < ActiveRecord::Migration[5.0]
+  def up
+    # create tables
+    create_table :tiers do |t|
+      t.string :name, null: false, unique: true
+      t.timestamps
+    end
+
+    create_table :service_tiers do |t|
+      t.integer :service_id
+      t.integer :tier_id
+      t.timestamps
+    end
+
+    add_column :local_authorities, :tier_id, :integer, index: true
+
+    # migrate data
+    unitary_tier = Tier.create(name: 'unitary')
+    district_tier = Tier.create(name: 'district')
+    county_tier = Tier.create(name: 'county')
+
+    LocalAuthority.all.each do |la|
+      la.tier_id = Tier.find_by(name: la.tier).id
+      la.save
+    end
+
+    Service.where.not(tier: nil).each do |service|
+      case service.tier
+      when 'district/unitary'
+        ServiceTier.create(service: service, tier: district_tier)
+        ServiceTier.create(service: service, tier: unitary_tier)
+      when 'county/unitary'
+        ServiceTier.create(service: service, tier: county_tier)
+        ServiceTier.create(service: service, tier: unitary_tier)
+      when 'all'
+        ServiceTier.create(service: service, tier: county_tier)
+        ServiceTier.create(service: service, tier: unitary_tier)
+        ServiceTier.create(service: service, tier: district_tier)
+      end
+    end
+
+    # delete old data
+    remove_column :local_authorities, :tier
```

Comment:
We should hold off deleting the old data until we're confident that everything works